pyro/distributions/torch_distribution.py

```diff
@@ -202,3 +213,50 @@ def mean(self):
     @property
     def variance(self):
         return self.base_dist.variance.expand(self.batch_shape + self.event_shape)
+
+
+class MaskedDistribution(TorchDistribution):
+    """
+    Masks a distribution by a zero-one tensor that is broadcastable to the
+    distributions ``batch_shape``.
+
+    :param Variable mask: A zero-one valued float tensor.
+    """
+    def __init__(self, base_dist, mask):
+        if broadcast_shape(mask.shape, base_dist.batch_shape) != base_dist.batch_shape:
+            raise ValueError("Expectewd mask.shape to be broadcastable to base_dist.batch_shape, "
```

Comment:
typo: Expectewd

Reply:
Fixed. ugh, now for another travis cycle.